content/en/docs/concepts/architecture/nodes.md

```diff
@@ -88,11 +92,19 @@ Enabling this feature creates a small delay between the
 time when a condition is observed and when a taint is created. This delay is usually less than one second, but it can increase the number of Pods that are successfully scheduled but rejected by the kubelet.
 {{< /caution >}}
 
-### Capacity
+### Capacity and Allocatable
```

Comment:
```suggestion
### Capacity and Allocatable {#capacity}
```
would keep the hyperlink for this section be the same, ie https://kubernetes.io/docs/concepts/architecture/nodes/#capacity